### DISCUSSION TITLE
Upgrade to Jakarta

### DISCUSSION THREAD

**Discussion by ivarwedoeskatteetaten** [difi/vefa-peppol#55] [General]
Hello!
Do you have any plans for when you will upgrade to Jakarta?
I see that SbdhHelper and other classes still use imports from javax.xml.bind.

**Comment by aaron-kumar** [comment]
@ivarwedoeskatteetaten Probably it will be part of H2, 2024. More clarity will be given in "Oxalis Community Business Meeting" on 20th March 2024

**Reply by ivarwedoeskatteetaten** [reply]
@aaron-kumar is there any updated info after that meeting you mentioned?
